engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html

```diff
@@ -283,4 +283,54 @@ <h5 translate ng-show="repository.content_type == 'docker'">Upload Docker Image<
     </form>
   </section>
 
+  <section ng-show="repository.content_type === 'yum'">
+    <div class="divider"></div>
+    <h4 translate>Distribution Information</h4>
```

Comment:
For things like centos, the main repo includes the kickstart content.  So in that case it may not be obvious.  I think it makes sense to show for all yum repos, which john has done here.